FinderKeeper/Assets/Mapbox/Examples/Scripts/RotateWithLocationProvider.cs

```diff
@@ -50,7 +50,7 @@ public class RotateWithLocationProvider : MonoBehaviour
 
 		Quaternion _targetRotation;
 
-		bool _isRotatable = true;
+		public bool _isRotatable = true;
```

Comment:
Just nitpicking, but if this is public now, the underscore should be removed

Reply:
Agreed. Fixed